civihr_employee_portal/civihr_employee_portal.module

```diff
@@ -1723,6 +1731,17 @@ function civihr_employee_portal_menu() {
   return $items;
 }
 
+/**
+ * Callback function for 'emergency_contacts/{id}/delete
+ *
+ * @param string $id
```

Comment:
Shouldn't this be an `int` instead?

Reply:
It should, but it's actually a string since it seems all these params are passed as a string. I'm pretty sure there are plenty of places like this where I _want_ it to be an int, like if I'm fetching some value using a BAO, but in fact it's a string so I think it's better to say what it is instead of what it should be so nobody does a strict comparison which would fail

Reply:
As you said, there are plenty of places like this. Unfortunately, is something caused by how badly the tools and libraries we use deal with data.

Given this is something out of our control, I think it's better for us to use the docblock to inform what we expect this to be, rather than what the library/framework/whatever will pass. I think it's better having a failing strict comparison, than someone thinking they could pass something like `foobar` to the function.